product_template_tags/README.rst

```diff
@@ -0,0 +1,71 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=====================
+Product Template Tags
+=====================
+
+This addon allow to add tags on products.
+
+Installation
+============
+
+Just install it!
```

Comment:
I think that such readme part is unnecessary, and could be removed. (not a blocking point)